ruby/ror/config/database.yml

```diff
@@ -0,0 +1,19 @@
+
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: postgres
+  username: postgres
```

Comment:
@rkarthik007 do we want to update database/username to something other?

Reply:
Thought about it, lets sweep through all the different orm's and normalize it afterwards. For now, this is ok.